projects/plugins/jetpack/modules/widgets/gallery.php

```diff
@@ -259,6 +259,7 @@ public function circle_widget( $args, $instance ) {
 	 * @return string String of HTML representing a slideshow gallery
 	 */
 	public function slideshow_widget( $args, $instance ) {
+		$attr = array();
```

Comment:
I see where we check `$attr` below but I don't see anywhere else where `$attr` gets initialized. I wonder if this is a bug in the original code.

Reply:
I saw this weird one... There is even a comment to make phpcs ignore this problem on line 304 and a TODO to fix it.
I thought this was beyond the scope of these changes, but what do you think @markbiek ?

Reply:
I'd ask in the jetpack-developers Slack channel and see what they say. If it is an actual bug, we might be able to handle the fix. Otherwise we can create an issue for the Jetpack team to handle it.